gpii/node_modules/preferencesServer/test/preferencesServerTests.js

```diff
@@ -580,11 +580,11 @@ var getFixtures = [
                 "gpii-default": {
                     "name": "Default preferences",
                     "preferences": {
-                        "http://registry.gpii.net/common/onscreenKeyboard": true,
                         "http://registry.gpii.net/common/initDelay": 0.120,
                         "http://registry.gpii.net/common/cursorSpeed": 0.850,
                         "http://registry.gpii.net/common/cursorAcceleration": 0.800,
                         "http://registry.gpii.net/common/mouseEmulationEnabled": true,
+                        "http://registry.gpii.net/common/onScreenKeyboardEnabled": true,
```

Comment:
@cstrobbe yeah - old though and all related to keyboard ... they were taken directly from ISO24751 and used to cover the available settings for the linux keyboard settings